includes/class-sensei-usage-tracking.php

```diff
@@ -69,6 +73,24 @@ protected function opt_in_dialog_text() {
 		);
 	}
 
+	protected function do_track_plugin( $plugin_slug ) {
+		if ( 1 === preg_match( '/^sensei/', $plugin_slug ) ) {
+			return true;
+		}
+		$third_party_plugins = array(
```

Comment:
A quick scan of Sensei and it looks looks like the third-party plugins it works with/would be interesting to track are:
- jetpack
- woocommerce
- woocommerce-subscriptions
- sitepress-multilingual-cms (WPML)
- wp-quicklatex

Could be more, but those are the ones I could see.